HealthcareSystem/PatientWebApp/wwwroot/js/filling_out_the_survey.js

```diff
@@ -1,7 +1,32 @@
 ﻿var params = (new URL(window.location.href)).searchParams;
 var id = params.get("id");
-
+var jmbg = "";
 $(document).ready(function () {
+	var token = window.localStorage.getItem('token');
+	if (token != null) {
+		$.ajax({
+			url: "/api/user/logged",
+			type: 'GET',
+			dataType: 'json',
+			processData: false,
+			contentType: 'application/json',
+			data: JSON.stringify(token),
+			success: function (loggedUser) {
+				if (loggedUser.role != "Patient") {
+					alert('Access denied!');
+					return;
+				}
+				jmbg = loggedUser.jmbg;
+			},
+			error: function () {
+				alert('Error getting logged user!');
+			}
+		});
+	}
+	else {
+		alert('Unlogged user!');
```

Comment:
Also, please remove calls to alert before the branch is merged. In these cases I feel like a redirect to the login page would be more appropriate anyway (since you're not supposed to be able to access these pages without being logged in.